web/src/app/forms/FormField.js

```diff
@@ -82,17 +82,34 @@ export function FormField(props) {
     ..._inputProps,
   }
 
-  let getValueOf = (e) => (e && e.target ? e.target.value : e)
+  // mutable function
+  let getValueOf = (e) => {
+    if (e && e.target && e.target.value) {
+      return e.target.value
+    }
```

Comment:
If the value is falsy, (like an empty input), this would return the event object instead of the value

Reply:
wouldn't it return `""` or `false`?

Reply:
The line below this returns `e` rather than `e.target`

We'd want this to be `if (e && e.target) return e.target.value`

We only return the `e` arg if it's not an object with a `.target` (which is admittedly confusing).

Reply:
gotcha, i see what you mean now. if the value that we want to set is `false` and is coming from a checkbox, this check would fail and then return `e` instead of the intended `e.target.value`. thanks!